source/_integrations/becker.markdown

```diff
@@ -0,0 +1,60 @@
+---
+title: Velux
+description: Instructions on how to integrate Becker Covers via Becker Centronic USB Strick with Home Assistant.
+ha_category:
+  - Cover
+ha_release: 0.110
+ha_codeowners:
+  - '@nicolasberthel'
+ha_domain: becker
+---
+
+Becker Cover Integration for for Home Assistant allows you to control your Becker Covers via a centronic USB Stick
+
+There is currently support for the following device types within Home Assistant:
+
+- Cover
+
+## Configuration
+
+The becker cover configuration consist in a new becker platform in your cover configuration:
+
+```yaml
+# Example configuration.yaml entry
+cover:
+  - platform: becker
```

Comment:
New integrations configured via a platform is no longer allowed.
Please adjust the integration. 👍

Reply:
Integration changed